skype/skype-ps/skype/New-CsTeamsAudioConferencingPolicy.md

```diff
@@ -0,0 +1,161 @@
+---
+external help file: MicrosoftTeams-help.xml
+Module Name: MicrosoftTeams
+online version: https://docs.microsoft.com/en-us/powershell/module/skype/new-csteamsaudioconferencingpolicy
+schema: 2.0.0
+---
+
+# New-CsTeamsAudioConferencingPolicy
+
+## SYNOPSIS
+
+## SYNTAX
+
+```
+New-CsTeamsAudioConferencingPolicy [-Identity] <String> [-AllowTollFreeDialin <Boolean>] [-Force] [-InMemory]
```

Comment:
Same here, what about MsftInternalProcessingMode?

Reply:
It's not exposed in TRPS.